crates/tabby/src/main.rs

```diff
@@ -134,6 +137,14 @@ async fn main() {
     init_logging(cli.otlp_endpoint);
 
     let config = Config::load().unwrap_or_default();
+    let root = tabby_common::path::tabby_root();
+    std::fs::create_dir_all(&root).expect("Must be able to create tabby root");
+    #[cfg(target_family = "unix")]
```

Comment:
is there an windows equivalent to 0600 for directory permission?

Reply:
I believe windows doesn't let you read other users' files by default. The only permission that can be modified without the unix extension is whether the file is readonly.